types/react-vis/index.d.ts

```diff
@@ -0,0 +1,3868 @@
+
+        getColor0?: (datapoint: number, info: infoObject) => void;
+
+        getOpacity?: (datapoint: number, info: infoObject) => void;
+
+        getOpacity0?: (datapoint: number, info: infoObject) => void;
+
+        getSize?: (datapoint: number, info: infoObject) => void;
+
+        getSize0?: (datapoint: number, info: infoObject) => void;
+
+        getX?: (datapoint: number, info: infoObject) => void;
+
+        getX0?: (datapoint: number, info: infoObject) => void;
+
+        getY?: (datapoint: number, info: infoObject) => void;
+
+        getY0?: (datapoint: number, info: infoObject) => void;
+
+        height?: number;
+
+        onNearestX?: (datapoint: object, info: infoObject) => void;
+
+        onNearestXY?: (datapoint: object, info: infoObject) => void;
+
+        onSeriesClick?: (datapoint: number, info: infoObject) => void;
+
+        onSeriesMouseOut?: (datapoint: number, info: infoObject) => void;
+
+        onSeriesMouseOver?: (datapoint: number, info: infoObject) => void;
+
+        onSeriesRightClick?: (datapoint: number, info: infoObject) => void;
+
+        onValueClick?: (datapoint: number, info: infoObject) => void;
+
+        onValueMouseOut?: (datapoint: number, info: infoObject) => void;
+
+        onValueMouseOver?: (datapoint: number, info: infoObject) => void;
+
+        onValueRightClick?: (datapoint: number, info: infoObject) => void;
+
+        opacityBaseValue?: number;
+
+        opacityDistance?: number;
+
+        opacityDomain?: Array<number>;
+
+        opacityRange?: Array<number>;
+
+        opacityType?: ('linear' | 'ordinal' | 'category' | 'time' | 'time-utc' | 'log' | 'literal');
+
+        sizeBaseValue?: number;
+
+        sizeDistance?: number;
+
+        sizeDomain?: Array<number>;
+
+        sizeRange?: Array<number>;
+
+        sizeType?: ('linear' | 'ordinal' | 'category' | 'time' | 'time-utc' | 'log' | 'literal');
+
+        stack?: boolean;
+
+        style?: Object;
+
+        width?: number;
+
+        xBaseValue?: number;
+
+        xDistance?: number;
+
+        xDomain?: Array<number | Date>;
+
+        xRange?: Array<number>;
+
+        xType?: ('linear' | 'ordinal' | 'category' | 'time' | 'time-utc' | 'log' | 'literal');
+
+        yBaseValue?: number;
+
+        yDistance?: number;
+
+        yDomain?: Array<number>;
+
+        yRange?: Array<number>;
+
+        yType?: ('linear' | 'ordinal' | 'category' | 'time' | 'time-utc' | 'log' | 'literal');
+
+    };
+
+    export type PropsRadarChart = {
+
+        className?: string;
+
+        colorRange?: string[];
+
+        colorType?: string;
+
+        hideInnerMostValues?: boolean;
+
+        startingAngle?: number;
+
+        animation?: { duration: number } | boolean;
+
+        data?: Array<Object>;
+
+        domains?: Array<Object>;
+
+        height?: number;
+
+        margin?: number | Object;
+
+        style?: Object;
+
+        tickFormat?: (tickValue: Date | number) => string | number;
+
+        width
```

Comment:
This doesn't seem to be used anywhere?

Reply:
I added this comment on line 3856 (`export type PropsmakeHeightFlexible`) and that's how it shows in the files view, a github bug must be making it show on line 2602 here.